.circleci/config.yml

```diff
@@ -82,3 +101,16 @@ workflows:
       - install-cni:
           requires:
             - build
+
+  # Nightly publish
+  nightly:
+    triggers:
+    - schedule:
+        cron: "30 * * * *"
+        filters:
+          branches:
+            only:
+            - master
+    jobs:
+    - nightly:
+        context: org-global
```

Comment:
I think we need to make this like istio/istio has it with the "nightly" job just doing the stuff required for docker push but the "nightly" workflow invoking the other jobs as well--lint, install-cni (test), etc.

```
    jobs:
      - lint
      - build
      - install-cni:
          requires:
            - build
        - nightly:
             context: org-global
             requires:
              - build
```

Reply:
Yes, or the other way around. Lint/build are separate in circle because they're very slow.